common-o2.sh

```diff
@@ -0,0 +1,48 @@
+package: Common-O2
+version: "%(tag_basename)s"
+tag: v0.2.0
+requires:
+  - boost
+  - "GCC-Toolchain:(?!osx)"
+build_requires:
+  - CMake
+source: https://github.com/AliceO2Group/Common
+incremental_recipe: |
+  make ${JOBS:+-j$JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/sh
+
+case $ARCHITECTURE in
+  osx*) BOOST_ROOT=$(brew --prefix boost) ;;
+esac
+
+cmake $SOURCEDIR                                              \
+      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                     \
+      -DCMAKE_PREFIX_PATH=$PROTOBUF_ROOT                      \
+      ${BOOST_ROOT:+-DBOOST_ROOT=$BOOST_ROOT}                 \
+      ${BOOST_ROOT:+-DBoost_DIR=$BOOST_ROOT}                  \
+      ${BOOST_ROOT:+-DBoost_INCLUDE_DIR=$BOOST_ROOT/include}  
+
+make ${JOBS+-j $JOBS} install
+
+#ModuleFile
+mkdir -p etc/modulefiles
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0 ${BOOST_VERSION:+boost/$BOOST_VERSION-$BOOST_REVISION} ${GCC_TOOLCHAIN_ROOT:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION}
```

Comment:
`GCC_TOOLCHAIN_ROOT` should be `GCC_TOOLCHAIN_VERSION` to be consistent with other recipes (the `BOOST_*` part is OK).